installation/proc-config-dynamic-plugins-rhdh-operator.adoc

```diff
@@ -0,0 +1,70 @@
+[id="proc-config-dynamic-plugins-rhdh-operator_{context}"]
+= Configuring dynamic plugins with the {product} Operator
+
+You can store the configuration for dynamic plugins in a ConfigMap object that the Custom Resource can reference.
```

Comment:
```suggestion
You can store the configuration for dynamic plugins in a ConfigMap object that the custom resource (CR) can reference.
```

In general, can we maybe start specifying which custom resource we're referring in procedures? As someone new to these docs, it's a bit confusing that it's not spelled out - I could see how it could be confusing for new administrators. It's not clear from this whether it's an existing configmap or one a user needs to create

Reply:
@abrennan89 I agree. Can we... create a separate Jira ticket for this? :)

Reply:
https://issues.redhat.com/browse/RHIDP-2255